client/src/components/ProductDetail/ProductDetail.js

```diff
@@ -65,7 +68,14 @@ const ProductContainer = styled.section`
   display: grid;
   gap: 10px;
   margin-bottom: 20px;
+
+  @media (min-width: 992px) {
+    grid-template-columns: 1fr 1fr;
+  }
 `
+
+const ProductContent = styled.div``
```

Comment:
If you don't need a custom styling, you should consider removing the rule and replacing the element with a simple div in the code above.

Reply:
good advoice, thanks